.ci/aarch64/filter_docker_aarch64.sh

```diff
@@ -33,6 +35,10 @@ filter_and_build()
 
 main()
 {
+	# install yq if not exist
+        if ! command -v yq >/dev/null; then
```

Comment:
yq is installed in the non standard PATH `${GOPATH_LOCAL}/bin/`, so maybe it's better to check for `${GOPATH_LOCAL}/bin/yq` ?

Reply:
but since CI on aarch64 is on bare-metal machine, there are possibilities that yq has already been installed in some other PATH, `/usr/bin`, `/usr/local/bin`, etc. ☹